cmd/requester.go

```diff
@@ -348,12 +233,16 @@ func requester(uri string, proxy string, useragent string) {
 	if len(h) < 4 {
 		uri += "/"
 	}
-	if len(useragent) == 0 {
-		useragent = "dontgo403/0.2"
+	if len(userAgent) == 0 {
+		userAgent = "dontgo403/0.2"
+	}
+
+	headers := []header{
+		{"User-Agent", userAgent},
 	}
```

Comment:
It makes it easier to now add [this](https://twitter.com/RoloMijan/status/1483943635062333441) header-related feature.